packages/swingset-theme-hashicorp/src/components/side-nav/index.tsx

```diff
@@ -1,20 +1,49 @@
 import { NavigationTree } from 'swingset/types'
 import Category from './category'
+import { Link } from '../link'
+import { cx } from 'class-variance-authority'
+import { ToggleButton } from './toggle-button'
 
 type SideNavBarProps = {
   categories: NavigationTree
+  isOpen: boolean
+  toggle: () => void
 }
 
-function SideNavigation({ categories }: SideNavBarProps) {
-  
+function SideNavigation({ categories, isOpen, toggle }: SideNavBarProps) {
+  const renderedCategories = categories.map((category) => (
+    <Category
+      title={category.title}
+      key={category.title}
+      items={category.children}
+    />
+  ))
 
-  const renderCategories = categories.map(
-    (category) => (
-      <Category title={category.title} key={category.title} items={category.children} />
-    )
-  )
+  return (
+    <>
+      <aside
+        aria-disabled={!isOpen}
```

Comment:
I believe we would want `aria-hidden` here, as we're visually hiding it from view. To make it not accessible by using `TAB`, we can also make sure the element has `visibility: hidden;` when it is closed (class `ss-invisible`).